packages/web/config/user-analytics-v2.ts

```diff
@@ -142,8 +143,17 @@ export const EventName = {
     depositAssetCompleted: "Deposit asset: Deposit completed",
     withdrawAssetStarted: "Withdraw asset: Withdraw started",
     withdrawAssetCompleted: "Withdraw asset: Withdraw completed",
-    buyOsmoClicked: "Assets: Buy OSMO clicked",
     buyOsmoStarted: "Assets: Buy OSMO started",
     buyOsmoCompleted: "Assets: Buy OSMO completed",
   },
+  // Events in profile modal
+  ProfileModal: {
```

Comment:
@victortokaito we'll need these to be added to amplitude

https://data.amplitude.com/osmosis-zone/Osmosis%20%F0%9F%A7%AA/events/main/latest